scripts/common.sh

```diff
@@ -548,6 +548,46 @@ configure_local_api_keys() {
     print_success "API keys configured successfully"
 }
 
+generate_litellm_master_key() {
+    # If CRS_KEY_ID is already set to the default value, skip key generation
+    if [ -f "deployment/env" ]; then
+        source deployment/env
+    fi
+    if [ "${LITELLM_MASTER_KEY:-}" != "" ] && [ "$LITELLM_MASTER_KEY" != "d5179c62ae1c7366e3ee09775d0993d5" ]; then
+        print_status "LITELLM_MASTER_KEY is already configured, skipping LiteLLM master key generation."
+        return 0
+    fi
+
+    print_linebreak
+    print_status "Configuring LiteLLM master key..."
+    local litellm_master_key=$(openssl rand -hex 16)
+    portable_sed "s|.*export LITELLM_MASTER_KEY=.*|export LITELLM_MASTER_KEY=\"$litellm_master_key\"|" deployment/env
+    print_success "LiteLLM master key configured successfully"
+}
+
+generate_crs_key_id_token() {
+    # If CRS_KEY_ID is already set to the default value, skip key generation
```

Comment:
It appears as we skip key generation if the `CRS_KEY_ID` is _not_ set to default value.